subprojects/docs/src/docs/userguide/jvm/building_java_projects.adoc

```diff
@@ -237,6 +240,8 @@ That's also how you can change the verbosity of the compiler, disable debug outp
 === Targeting a specific Java version
 
 By default, Gradle will compile Java code to the language level of the JVM running Gradle.
+With the usage of <<toolchains.adoc#,Java toolchains>>, you can break that link by making sure a given Java version, defined by the build, is used for compilation, execution and documentation.
+Sometimes however, what you need requires more advanced options.
```

Comment:
Sometimes however, what you need requires more advanced options.

:o: This sentence is a bit unclear. What advanced options? What "sometimes"?

What about something like:

> It is however possible to override some compiler and execution options at the task level.

Reply:
Much better indeed, thanks!